pkg/kubelet/volumemanager/metrics/metrics.go

```diff
@@ -20,15 +20,30 @@ import "github.com/prometheus/client_golang/prometheus"
 
 func init() {
 	prometheus.MustRegister(volumeManagerVolumesMetric)
+	prometheus.MustRegister(reconstructVolumeErrorsMetric)
 }
 
-var volumeManagerVolumesMetric = prometheus.NewGaugeVec(
-	prometheus.GaugeOpts{
-		Name: "volume_manager_total_volumes",
-		Help: "Number of volumes in Volume Manager",
-	}, []string{"state"})
+var (
+	volumeManagerVolumesMetric = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "volume_manager_total_volumes",
+			Help: "Number of volumes in Volume Manager",
+		}, []string{"state"})
+
+	reconstructVolumeErrorsMetric = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "volume_manager_reconstruct_volume_errors_total",
+			Help: "Amount of errors on ReconstructVolumeOperation",
+		},
+		[]string{"volume_name"})
```

Comment:
You should not use `volume_name` as label, there can be too many of them. From https://prometheus.io/docs/practices/naming/: 

> Do not use labels to store dimensions with high cardinality (many different label values), such as user IDs, email addresses, or other unbounded sets of values.

Reply:
Gotcha, thanks. I'll remove it.

Reply:
I could be useful to have plugin name as a label, if the plugin name is easily available.